controllers/automation/policyautomation_controller.go

```diff
@@ -464,7 +507,12 @@ func (r *PolicyAutomationReconciler) Reconcile(
 							// new event from compliant to non-compliant
 							eventMap[clusterName] = clusterEvent
 						}
-					} // Otherwise, the policy keeps non-compliant since originalStartTime, do nothing
+					} else {
+						// Otherwise, the policy keeps non-compliant since originalStartTime, do nothing
+						if !trimmedTargetMap[clusterName] {
```

Comment:
Could you please explain this change a bit more?

Reply:
without this, it creates one by one, in other words, it creates 2-3 ansible jobs when we have 3 policies(clusters) changed. also our ticket 
This is their expectation so 
Expected results:
All violations status of all clusters should be passed